dashboard/templates/dashboard.html

```diff
@@ -102,8 +108,8 @@
         </td>
         <td>
         {% set openqa_summary = project.openqa_summary() %}
-        {% for state in openqa_summary.keys() %}
-          {{ state }}: {{ openqa_summary[state]|length }}
+        {% for state in openqa_summary.keys()|sort %}
```

Comment:
alphabetical sort of the openQA states  is not in our interest

Reply:
hmm... it looked like random order without this. In the screenshot above you can see `failed` before `softfailed` in the ARM line and the other way in the Factory line. Or is this order provided in some other way?

Reply:
when ordering, then at a specified order which helps 'more' than alphabetical I'd say (passed, softfailed, failed, parallel_failed, cancelled, user_canceller)

'passed' as first is interestingly enough constant throughout the dashboard even without sorting

Reply:
IMO sorted in any order is an improvement over random